### ISSUE TITLE
Assuming this is dead. Alternative?

### ISSUE THREAD

**Issue by detoro84** [therealglazou/bluegriffon#101]
I love the functionalities but on Big Sur is unusable... cannot see the menus (they do not render) and lastly "undo" didn't work and lost all the work.
Any reliable alternative?
Thanks

**Comment by benwiggy** [comment]
I'd like to think that someone must be able to fix the UI problems on the Mac, rather than throwing all this work away. It's bizarre how few decent WYSIWYG/code web editors there are.

**Comment by Reaper10** [comment]
@benwiggy some one should take this code base and start over so every one can use on any os for 100% free, Wich make it open source to alterative open element. Witch is not  open source.

**Comment by chikega** [comment]
> I love the functionalities but on Big Sur is unusable... cannot see the menus (they do not render) and lastly "undo" didn't work and lost all the work. Any reliable alternative? Thanks

It's sad to see. It still works on Windows 11. But there are other alternatives, one of which is PineGrow.

**Comment by benwiggy** [comment]
Yes, PineGrow looks like the only decent option. It's around 2.5 Adobe months (!) for one year's subscription; or 3.75 Adobe months to purchase outright, with 50% off that for updates. So updating every 2 years would be slightly cheaper than the sub, I think.

Frankly, I think it's worth it.

**Comment by chikega** [comment]
I find that PineGrow has a very adaptable and fluid interface since it's built web technology (HTML/CSS/ NW.js/Electron) vs a GUI based on QT(C++) that Adobe and some other companies employ. It also ties in well with Visual Code and Atom. And the developers are very forward thinking implementing such features as a visual editor for Tailwind CSS.

**Comment by vehosah** [comment]
> > I love the functionalities but on Big Sur is unusable... cannot see the menus (they do not render) and lastly "undo" didn't work and lost all the work. Any reliable alternative? Thanks
> 
> It's sad to see. It still works on Windows 11. But there are other alternatives, one of which is PineGrow.

Yeah but there is a difference, it's paid

**Comment by ijeffsc** [comment]
I wonder if anyone has considered raising a bounty $$ on the source that 
is available to update Mac OS and/or other binaries.
